x-pack/test/alerting_api_integration/common/fixtures/plugins/actions_simulators/server/servicenow_simulation.ts

```diff
@@ -188,3 +230,19 @@ const handler = async (request: http.IncomingMessage, response: http.ServerRespo
   response.setHeader('Content-Type', 'application/json');
   response.end('Not supported endpoint to request servicenow simulator');
 };
+
+const getData = async (request: http.IncomingMessage) => {
+  let data: Record<string, unknown> = {};
+
+  if (request.method === 'POST') {
+    const buffers = [];
+
+    for await (const chunk of request) {
```

Comment:
if you attempt to `await` the request twice it'll fail with a promise rejection.